packages/react-utilities/src/hooks/useUnmount.test.tsx

```diff
@@ -17,7 +18,9 @@ describe('useUnmount', () => {
     expect(onUnmount).toBeCalledTimes(0);
     const wrapper = mount(<TestComponent />);
     expect(onUnmount).toBeCalledTimes(0);
-    wrapper.unmount();
+    ReactTestUtils.act(() => {
```

Comment:
I'll make  PR to react-utilities so this isn't required. we shouldn't be using enzyme in v9

Reply:
PR here: #21060

Reply:
Thanks! I assume both the hook and the test were a direct copy-paste from v8.